tests/mutex/LockMutexTest.php

```diff
@@ -106,4 +104,41 @@ public function testUnlockFailsAfterException()
             throw new \DomainException();
         });
     }
+
+    /**
+     * Tests the code result is available in LockReleaseException.
+     */
+    public function testCodeResultAvailableAfterFailedUnlock()
+    {
+        $this->mutex->expects($this->once())
+            ->method("unlock")
+            ->willThrowException(new LockReleaseException());
+
+        try {
+            $this->mutex->synchronized(function () {
+                return "result";
+            });
+        } catch (LockReleaseException $exception) {
+            $this->assertEquals("result", $exception->getCodeResult());
```

Comment:
Please assert that the getCodeException() is `null`.

Reply:
Added.